integration-tests/src/test/groovy/com/okta/sdk/tests/it/EventHooksIT.groovy

```diff
@@ -0,0 +1,254 @@
+/*
+ * Copyright 2020-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.sdk.tests.it
+
+import com.okta.sdk.resource.ResourceException
+import com.okta.sdk.resource.EventHookChannel
+import com.okta.sdk.resource.EventHookChannelConfig
+import com.okta.sdk.resource.EventHookChannelConfigAuthScheme
+import com.okta.sdk.resource.EventHookChannelConfigHeader
+import com.okta.sdk.resource.EventSubscriptions
+import com.okta.sdk.resource.event.hook.EventHook
+import com.okta.sdk.resource.event.hook.EventHookChannelConfigAuthSchemeType
+import com.okta.sdk.resource.event.hook.EventHookList
+import com.okta.sdk.tests.it.util.ITSupport
+import org.testng.annotations.Test
+
+import static org.hamcrest.MatcherAssert.assertThat
+import static org.hamcrest.Matchers.equalTo
+import static org.hamcrest.Matchers.hasSize
+import static org.hamcrest.Matchers.notNullValue
+
+/**
+ * Tests for /api/v1/eventHooks
+ * @since 2.0.0
+ */
+class EventHooksIT extends ITSupport {
+
+    @Test
+    void createEventHookTest() {
+        String name = "event-hook-java-sdk-${UUID.randomUUID().toString()}"
+
+        EventHook createdEventHook = client.instantiate(EventHook)
+            .setName(name)
+            .setEvents(client.instantiate(EventSubscriptions)
+                .setType(EventSubscriptions.TypeEnum.EVENT_TYPE)
+                .setItems(["user.lifecycle.create", "user.lifecycle.activate"]))
+            .setChannel(client.instantiate(EventHookChannel)
+                .setType(EventHookChannel.TypeEnum.HTTP)
+                .setVersion("1.0.0")
+                .setConfig(client.instantiate(EventHookChannelConfig)
+                    .setUri("https://www.example.com/eventHooks")
+                    .setHeaders([ client.instantiate(EventHookChannelConfigHeader)
+                                      .setKey("X-Test-Header")
+                                      .setValue("Test header value")])
+                    .setAuthScheme(client.instantiate(EventHookChannelConfigAuthScheme)
+                .setType(EventHookChannelConfigAuthSchemeType.HEADER)
+                .setKey("Authorization")
+                .setValue("Test-Api-Key"))))
```

Comment:
```suggestion
                    .setType(EventHookChannelConfigAuthSchemeType.HEADER)
                    .setKey("Authorization")
                    .setValue("Test-Api-Key"))))
```
nit: looks like these should be nested under config

Reply:
Idea: there is a lot of room for a "builder" helper class here.  There is a lot of static data needed to create a webhook that never changes (the channel, versions, etc).  All that is needed is:

- url
- auth (header/value)
- additional headers (optional)

Maybe something like:

```java
EventHookBuilder.instance()
    .setUrl("https://www.example.com/eventHooks")
    .setAuthorizationHeader("Test-Api-Key")
    .setHeaders(Map.of("X-Test-Header",  "Test header value"))
    .buildAndCreate(client);
```

Reply:
Will add a builder (aka lifesaver) :)